src/service/stellar.ts

```diff
@@ -25,8 +27,9 @@ export async function findNativeBalance(signerAccountPublicKey: string): Promise
 
 export async function loadTransactionHistory(
   signerAccountPublicKey: string,
-): Promise<ServerApi.OperationRecord[]> {
+): Promise<Transaction[]> {
   try {
+    const transactionList: Transaction[] = [];
```

Comment:
`transactions`. It is unnecessary to clarify that this is a list because we already know that by declaring the variable there will be an array inside.